golem-worker-service-base/src/repo/api_definition_repo.rs

```diff
@@ -245,6 +275,31 @@ where
         }
     }
 
+    async fn make_immutable(
```

Comment:
maybe  `make_immutable` -> `set_not_draft`

Reply:
I considered it, but I believe that `make immutable` better represents intention while `set_not_draft` describes implementation details.

Reply:
i am not sure if this is implementation detail, 

basically there is `draft` flag in Api definition which at some point is synonym for immutable,
at the other side there is use of `immutable` , for example  `ApiRegistrationRepoError::Immutable` 

```rust
#[error("Immutable: ApiDefinition with id: {} and version: {} in namespace {} can not be updated", .0.id, .0.version, .0.namespace)]
    Immutable(ApiDefinitionKey<String>),
```
and then solution working with two words which more like mean same, 
and then this will require additional explanation what is relation between `draft` and `immutable`

if `immutable` state was something which is result of some computation or cover combination of multiple values or flags, then it could make sense to use different words

but otherwise, it is better to use just one term or word

Reply:
Done